var/spack/repos/builtin/packages/py-pyqt6/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyqt6(SIPPackage):
```

Comment:
Was planning on using `PythonPackage` but couldn't figure out how to pass `--confirm-license` to the build.

Reply:
:thinking: